python/ray/serve/_private/logging_utils.py

```diff
@@ -517,3 +520,155 @@ def __enter__(self):
     def __exit__(self, et, ev, tb):
         if self.level is not None:
             self.logger.setLevel(self.old_level)
+
+
+class ServeEventSummarizer:
```

Comment:
```suggestion
class ServeAutoscalingEventSummarizer:
```

Reply:
i think this class should be moved to its own file and not be part of `logging_utils`, its responsibility doesn't seem too related to what `logging_utils`  does